Makefile

```diff
@@ -82,8 +82,14 @@ $(extension_node_modules): node_modules
 $(extension_dists): src/extensions/npm/extensions/dist
 	cd $(@:/dist=) && ../../node_modules/.bin/npm run build
 
+.PHONY: clean-old-extensions
+clean-old-extensions:
+ifndef CI
+	find extensions -d 1 -type d '!' -exec test -e '{}/package.json' \; -delete
```

Comment:
Why do we need to skip this on CI? Is this incompatible with GNU `find`? How about Windows?

Reply:
Let me check this on my ubuntu VM.

In theory this doesn't need to run on CI ever, which is why I went with this solution, since this is really a developer helper when switching branches (which CI never does).

Reply:
Okay so GNU find and BSD find are not compatible with these options. I have found a new version that should work.